File: trunk/quests/kodtaz/#Hexxt_Chaos_Provoker.pl
sub EVENT_SPAWN {
  quest::settimer(81, 6);
}

sub EVENT_TIMER {
  if ($z > -437 && $timer == 81) {
    $npc->SetHP($npc->GetMaxHP());
  }
}

sub EVENT_DEATH_COMPLETE {
  quest::stoptimer(81);
  quest::depop(293212);
  quest::spawn2(293113,0,0,591,746,-460,75);
}

File: kodtaz/#Hexxt_Chaos_Provoker.lua
function event_spawn(e)
	eq.set_timer("1", 6000);
end

function event_timer(e)
	if (e.self:GetZ() > -437) then
		e.self:SetHP(e.self:GetMaxHP());
	end
end

function event_death_complete(e)
	--signal Trigger_Kodtaz to tell him I'm dead
	eq.signal(293218,3);
end
